laundry/data/laundry_data.csv

```diff
@@ -0,0 +1,53 @@
+0,Bishop White,Quad,5faec7e9-a4aa-47c2-a514-950c03fac460

```

Comment:
I think that it could make sense to have a LaundryMachine model that contains all of these fields. That way the machines are stored in our DB and user's favorited machines can be a direct link to a LaundryMachine. Do you guys think that makes sense? We should definitely talk more about this